test/unit/core.js

```diff
@@ -286,13 +286,19 @@ QUnit.test( "type for `Symbol`", function( assert ) {
 });
 
 QUnit.asyncTest( "isPlainObject", function( assert ) {
-	assert.expect( 15 );
+	assert.expect( 17 );
 
-	var pass, iframe, doc,
+	var pass, iframe, doc, parentObj, childObj,
 		fn = function() {};
 
 	// The use case that we want to match
 	assert.ok( jQuery.isPlainObject( {} ), "{}" );
+	assert.ok( jQuery.isPlainObject( new Object() ), "new Object" ); // jshint ignore:line
```

Comment:
Because of `new Object()`, I guess. Is that test necessary considering that it's equivalent to `{}` if one doesn't modify `Object.prototype` which we don't support?